tool/tbot/main.go

```diff
@@ -88,6 +87,9 @@ func Run(args []string, stdout io.Writer) error {
 	startCmd.Flag("join-method", "Method to use to join the cluster. "+joinMethodList).EnumVar(&cf.JoinMethod, config.SupportedJoinMethods...)
 	startCmd.Flag("oneshot", "If set, quit after the first renewal.").BoolVar(&cf.Oneshot)
 	startCmd.Flag("diag-addr", "If set and the bot is in debug mode, a diagnostics service will listen on specified address.").StringVar(&cf.DiagAddr)
+	startCmd.Flag("log-format", "Controls the format of output logs. Can be `json` or `text`. Defaults to `text`.").
```

Comment:
Would it make sense to add a global flag, like `debug`?

```suggestion
	app.Flag("log-format", "Controls the format of output logs. Can be `json` or `text`. Defaults to `text`.").
```

Reply:
The reason I'm avoiding this for now is that `tsh` doesn't support JSON logging yet and a number of the other commands invoke `tsh` internally - until I can propagate that setting, I don't want to misleadingly offer it for those commands.